README.md

```diff
@@ -31,8 +31,8 @@ If your collection is not present on the Ansible forum yet, please check out the
 -->
 
 * Join the Ansible forum:
-    * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
-    * [Posts tagged with 'your tag'](https://forum.ansible.com/tag/YOUR_TAG): subscribe to participate in collection-related conversations.
+    * [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example the `YOUR TAG` tag.
+    * [Posts tagged with 'your tag'](https://forum.ansible.com/tag/YOUR_TAG): subscribe to participate in collection/technogoly-related conversations.
```

Comment:
Typo

Reply:
Oops. I'll push a commit to `main` fixing that.

Reply:
be361bcd32507a0e81b3765772b6a45dcb792616